packages/runtime/src/components/builtin/Image/Image.tsx

```diff
@@ -125,22 +125,20 @@ const ImageComponent = forwardRef(function Image(
     boxShadow,
     className,
   )
-  const unoptimizedImageClassName = useStyle({ width: '100%' })
 
   if (!dimensions) return null
 
   return (
     <Container link={link} ref={ref} id={id} className={containerClassName}>
-      {isInBuilder ? (
-        <img className={unoptimizedImageClassName} src={imageSrc} alt={altText} />
-      ) : nextMajorVersion < 13 ? (
+      {nextMajorVersion < 13 ? (
         <NextLegacyImage
           layout="responsive"
           src={imageSrc}
           sizes={imageSizes(width)}
           alt={altText}
           width={dimensions.width}
           height={dimensions.height}
+          unoptimized={isInBuilder}
         />
```

Comment:
The reason why I changed `isInBuilder` yesterday was because of an SSR hydration error because we're rendering just a `<img/>` instead of `NextImage`.

When I looked into the commit why we're not using `NextImage` in the builder, it just says:
> Avoid using next/image in the builder

Can we still use `NextImage`, if we're using [`unoptimized` prop](https://nextjs.org/docs/api-reference/next/image#unoptimized)? @migueloller

Reply:
Yeah, that works